docs/reference/variables.md

```diff
@@ -0,0 +1,12 @@
+For ease of searching, an overview of all available variables is shown below. These can also be displayed using the `name` argument, for more information look [here](../formulating_requests.md#using-the-info-command).
+
+???+ tip
+    With the built-in `grep` command, you can also search through the lists of different variables in era5cli, for example:
+
+    ```
+    era5cli info "2Dvars" | grep temperature
+    ```
+
+    will list all single level variables that contain the word 'temperature'.
+
+### Lists of variables
```

Comment:
Where does the content come from?

Also: content looks like a code block. Might be helpful to include the command in the code block, e.g. 

```bash
$ era5cli --help
positional arguments:
  {hourly,monthly,info,config}
                        sub-command
```

Reply:
By the way, this is a really nice overview of the available options, and here the content tabs works perfectly!